src/main/java/edu/greenblitz/robotName/subsystems/swerve/Chassis/ChassisConstants.java

```diff
@@ -0,0 +1,27 @@
+package edu.greenblitz.robotName.subsystems.swerve.Chassis;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+public class ChassisConstants {
+
+    static final Pose2d INITIAL_ROBOT_POSITION = new Pose2d(0, 0, new Rotation2d(0));
+    public static final Translation2d[] SWERVE_LOCATIONS_IN_SWERVE_KINEMATICS_COORDINATES = new Translation2d[]{
+            //the WPILib coordinate system is stupid. (x is forwards, y is leftwards)
+            //the translations are given rotated by 90 degrees clockwise to avoid coordinates system conversion at output
+            new Translation2d(0.3020647, 0.25265), /*fl*/
+            new Translation2d(0.3020647, -0.25265),/*fr*/
+            new Translation2d(-0.3020647, 0.25265),/*bl*/
+            new Translation2d(-0.3020647, -0.25265)/*br*/};
+
```

Comment:
i think that the magic numbers are ok because its the module coordinates. if we will change the chassis maybe mechanics says that they want one module further away from the others so i think it is okay to use the duplicated magic numbers

Reply:
I don't see a likely scenario where out modules aren't located the same distance from each other.

If that does happen, you can directly use the numbers. Since that's not the case (and probably never will be), I think it's better to define the distances once and reuse their values with negative signs where needed.

Reply:
so do you think that i should call these variables, FRONT_LEFT_MODULE_X and FRONT_LEFT_MODULE_Y?
and than play with the -?

Reply:
Not 100% sure about the names but yeah, I think it will be much more readable and clean